devops/yarn/deploy/run.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
```

Comment:
Whether we need the sub directory deploy, is it ok just put the file of run.sh under the "devops/yarn" directory?